conf/routes

```diff
@@ -0,0 +1,17 @@
+# Routes
+# This file defines all application routes (Higher priority routes first)
+# ~~~~
+
+GET         /                     io.pathfinder.controllers.ViewController.index
```

Comment:
I'm referring specifically to the index part. I dont think we are hosting a webpage on this server.